script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_1D.txt

```diff
@@ -29,11 +29,11 @@ Now, there is no need to do so. There is no need to go anywhere.
 People are as free as they can be, but they also have no reason to step out of their narrow boxes.
 I think that's what the depressing sense of stagnation in classrooms, schools, and cities is all about.
 <Page9>
-And so on. Although I have tried to analyze this in a pretentious manner, these are just my own impressions.
-I do not feel a sense of stagnation.
+And so on. Although I tried to analyze this in a pretentious manner, these are just my own impressions.
+Persoanlly, I don't feel a sense of stagnation. //I think this helps split his thoughts from the random Nasu monologue in the last few pages, but it's a little redundant as well.
```

Comment:
`Personally` typo

Reply:
I should probably type my changes in Word or something else with a spell checker...